src/Microsoft.AspNetCore.Server.Kestrel.Core/Features/IHttpMinRequestBodyDataRateFeature.cs

```diff
@@ -6,7 +6,7 @@ namespace Microsoft.AspNetCore.Server.Kestrel.Core.Features
     /// <summary>
     /// Represents a minimum data rate for the request body of an HTTP request.
     /// </summary>
-    public interface IHttpRequestBodyMinimumDataRateFeature
+    public interface IHttpMinRequestBodyDataRateFeature
```

Comment:
After thinking about this for a little bit, I think we should change all usages of "Minimum" in public APIs to "Min" for consistency considering how many limits already use "Max".

So that means the MinimumDataRate class changes to MinDataRate, IHttpMinRequestBodyDataRateFeature.MinimumDataRate changes to IHttpMinRequestBodyDataRateFeature.MinDataRate, etc...

I don't care as much about private/internal names, but I think some of that could also change to match.